helm/acapy-cloud/conf/local/did-resolver.yaml

```diff
@@ -0,0 +1,38 @@
+fullnameOverride: did-resolver
+
+replicaCount: 1
+
+podAnnotations:
+  sidecar.istio.io/proxyCPU: 10m
+
+image:
+  registry: ghcr.io
+  name: cheqd/did-resolver
+  pullPolicy: Always
+  tag: latest
```

Comment:
Waiting for the next release of `cheqd/did-resolver` - https://github.com/cheqd/did-resolver/pull/363